Swiftz/Operators.swift

```diff
@@ -130,6 +138,16 @@ infix operator <+> {
 	associativity right
 }
 
+/// MARK: Data.JSON
+
+/// Retrieve | Retrieves a value from a dictionary of JSON values using a given keypath.
+///
+/// If the given keypath is not present or the retrieved value is not of the appropriate type, this
+/// function returns `.None`.
+infix operator <! {
```

Comment:
I'm open to ideas on this one.  Aeson uses `.:`, which Swift throws a fit at.  In fact, any permutation of dots and colons is a no go.  Maybe `<?` to emphasize this is the total version of retrieve and use `<!` as a version that forces the Optional?